airbyte-integrations/connectors/source-cockroachdb/src/test/java/io/airbyte/integrations/source/cockroachdb/CockroachDbJdbcSourceAcceptanceTest.java

```diff
@@ -188,6 +188,26 @@ protected List<AirbyteMessage> getTestMessages() {
                         COL_UPDATED_AT, "2006-10-19T00:00:00Z")))));
   }
 
+  @Test
+  protected void testDiscoverWithNonCursorFields() throws Exception {
```

Comment:
Here is UI screenshot there we can see that rowid is always present if PK had not been introduced for a table,
![co](https://user-images.githubusercontent.com/22977087/180973486-bdcb8a09-871f-4d94-973f-f7fcd7c29285.png)